### DISCUSSION TITLE
Ligature component retrieval

### DISCUSSION THREAD

**Discussion by kellyo-cricut** [harfbuzz/harfbuzz#4516] [Q&A] [answered]
I'm trying to use the `hb_ot_layout_lookup_collect_glyphs` call on a font with `liga` and `dlig` features (LookupType 4) in order to retrieve the ligature glyph ids and their components. Is this currently supported?  The output of that call is giving me some glyph ids, but not enough to map the `glyphs_input` values to corresponding values in `glyphs_output`.

**Comment by behdad** [comment, marked as answer]
No. We currently don't have API querying the ligatures.
